FRW/Modules/MultiBackup/Manager/Target/MultiBackupDropboxTarget.swift

```diff
@@ -0,0 +1,208 @@
+//
+//  MultiBackupDropboxTarget.swift
+//  FRW
+//
+//  Created by cat on 12/17/24.
+//
+
+import Foundation
+import UIKit
+import SwiftyDropbox
+
+// MARK: - MultiBackupDropboxTarget
+
+final class MultiBackupDropboxTarget: MultiBackupTarget {
+    // MARK: Lifecycle
+
+    init() {
+        if DropboxOAuthManager.sharedOAuthManager == nil {
+            let appKey = ServiceConfig.shared.dropboxAppKey
+            DropboxClientsManager.setupWithTeamAppKey(appKey)
+        }
+        path = "/" + MultiBackupManager.backupFileName
+        DropboxClientsManager.unlinkClients()
+    }
+
+    deinit {
+        if let observer = observer {
+            NotificationCenter.default.removeObserver(observer)
+        }
+    }
+
+    // MARK: Internal
+
+    var uploadedItem: MultiBackupManager.StoreItem?
+
+    var registeredDeviceInfo: SyncInfo.DeviceInfo?
```

Comment:
nit: if these properties are read, but not changed, outside of this class, then make them `private(set)`.

Reply:
First of this way